content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,37 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based sampling
+
+In order to ensure complete log information related to a given subset of transactions, sample on a specific attribute holding an ID, such as User, Session, Request Identifiers.
+By default, the log-based sampling is applied (exclusion percentage set on `all logs`).
+
+This feature operates on attribute's values rather than the log itself, for example:
+* given an attribute `id` holding 3 distinct values (`1`, `2`, `3`),
+* defining a 66% exclusion filter on it,
+* results in indexing all logs holding 1 specific attribute's value (`3`), all the other logs won't be indexed.
+
+{{< img src="logs/indexes/index_exclusion_attribute_groups.png" alt="index attribute-based exclusion filter groups"  style="width:70%;">}}
+
+For example, exclude `nginx` logs for 99% of `http.request_id`:
+
+{{< img src="logs/indexes/index_exclusion_on_attribute.png" alt="index attribute-based exclusion filter"  style="width:70%;">}}
+
```

Comment:
As discussed, let's focus on the http.request_id example and the trace one and put the emphasis on the fact that the kept traces are complete (which is the key functionality of this new sampling mechanism).

Reply:
Yes, `session_id` and `trace_id` examples only.
I prefer session instead of request as the latter has no concept of transaction unless we define an exclusion filter with a `*` query (on every log).